vpr/src/route/rr_graph.cpp

```diff
@@ -690,6 +689,18 @@ static void build_rr_graph(const t_graph_type graph_type,
     /* END OPIN MAP */
 
     bool Fc_clipped = false;
+    /* Draft the switches as internal data of RRGraph object
```

Comment:
@tangxifan 
Can you kindly Have a look here are these lines correctly  inserted here?

Reply:
Look good